cdap-distributions/src/hdinsight/install.sh

```diff
@@ -15,30 +15,18 @@
 # the License.
 
 #
-# Install CDAP for Azure HDInsight cluster
+# Install CDAP for Azure HDInsight v2 marketplace
 #
 
-# Arguments supplied by https://raw.githubusercontent.com/hdinsight/Edge-Node-Scripts/release-12-1-2015/edgeNodeSetup.sh
-CLUSTERNAME=${1}
-USERID=${2}
-PASSWD=${3}
-CUSTOMPARAMETER=${4} # Not used
-SSHUSER=${5}
-
-# Ambari constants
-AMBARICONFIGS_SH=/var/lib/ambari-server/resources/scripts/configs.sh
-AMBARIPORT=8080
-
-
 # CDAP config
 # The git branch to clone
 CDAP_BRANCH='release/3.4'
 # Optional tag to checkout - All released versions of this script should set this
 CDAP_TAG=''
 # The CDAP package version passed to Chef
-CDAP_VERSION='3.4.0-1'
+CDAP_VERSION='3.4.3-1'
 # The version of Chef to install
-CHEF_VERSION='12.7.2'
+CHEF_VERSION='12.10.24'
```

Comment:
``12.11.18`` breaks due to chef-zolo... ``12.11.17`` wasn't available, so went down to previous minor version